src/main/java/org/eu/ingwar/tools/arquillian/extension/suite/DeploymentClassFinder.java

```diff
@@ -30,6 +29,7 @@
 import java.util.Set;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
is this import correct? I do not see anything using `StringUtils` in this commit

Reply:
Yes.  Line 122 is the usage. Switched from old commons.lang to new commons.lang3